containers/wordpress/wp_base/http/files/1s-bkm.j2

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files
+# (the "Software"), to deal in the Software without restriction,
+# including without limitation the rights to use, copy, modify, merge,
+# publish, distribute, sublicense, and/or sell copies of the Software,
+# and to permit persons to whom the Software is furnished to do so, 
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES
+# OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE
+# OR OTHER DEALINGS IN THE SOFTWARE.
+#
+# SPDX-License-Identifier: MIT
+# this is only for the mysqld standalone daemon
+[mysqld]
+# Disabling symbolic-links is recommended to prevent assorted security risks
+symbolic-links=0
+
+open_files_limit=4000
+query_cache_limit=128K
+query_cache_size=1M
+query_cache_type=1
+#enable-networking
+bind-address = 0.0.0.0
+#Fine tuning
+key_buffer              = 16M # TODO: This should actually be key_buffer_size
+max_allowed_packet      = 16M
+thread_stack            = 192K
+thread_cache_size       = 8
+#thread_handling=pool_of_threads
+skip-external-locking
+key_buffer_size = 384M
+max_allowed_packet = 1M
+table_open_cache = 512 # TODO: Find the optimal value for this
+sort_buffer_size = 2M
+read_buffer_size = 2M
+read_rnd_buffer_size = 8M
+myisam_sort_buffer_size = 64M
+
+#innodb_buffer_pool_size = 1G #TODO: Find the optimal value for this (at least 1GB so we can have multiple instances)
+#innodb_buffer_pool_instances = 8
+#innodb_thread_concurrency = 80
+#innodb_adaptive_hash_index = OFF
+# The thread_concurrency is deprecated, no need to use it
+# thread_concurrency = 72
+# Increase number of maximum connections (used by oss-performance)
+max_connections        = 1000
+# Make sure you have query logging OFF or you will bump into low cpu utilization
+# and performance will drop
+#general-log
+#general-log-file=/var/log/mariadb/queries.log
+#log-output=file
+# This replaces the startup script and checks MyISAM tables if needed
+# the first time they are touched
+myisam-recover         = BACKUP
+
+#table_cache            = 64
```

Comment:
Why do we need to keep the commented entries such as this one and others? My suggestion is to remove them.